src/main/java/com/example/tournamentMaker/excel/football/FootballStatisticWriterService.java

```diff
@@ -0,0 +1,176 @@
+    private static final int PLACE_COLUMN_EXCEL_WIDTH = 3;
+    private static final int PLAYER_COLUMN_EXCEL_WIDTH = 26;
+    private static final int TEAM_COLUMN_EXCEL_WIDTH = 20;
+    private static final int STATISTIC_COLUMN_EXCEL_WIDTH = 12;
+
+
+    void writeStatistic(Workbook workbook, Sheet sheet, int rowStart, int colStart, Tournament tournament) {
+        ExcelUtil.createRows(sheet, MAX_ROWS_AMOUNT);
+        CellStyle borderCellStyle = ExcelUtil.createCellStyle(workbook, IndexedColors.GOLD.getIndex());
+        CellStyle fillingTableCellStyle = ExcelUtil.createCellStyle(workbook, IndexedColors.LEMON_CHIFFON.getIndex());
+
+        List<Team> teamList = tournament.getTeamList();
+        int secondTableColStart = colStart + STATISTIC_TABLE_WIDTH_WITH_FREE_SPACE;
+
+        createStatisticTable(sheet, rowStart, colStart, teamList, FootballStatisticType.GOALS,
+                borderCellStyle, fillingTableCellStyle);
+        createStatisticTable(sheet, rowStart, secondTableColStart, teamList, FootballStatisticType.YELLOW_CARDS,
+                borderCellStyle, fillingTableCellStyle);
+
+        rowStart += STATISTIC_TABLE_LENGTH_WITH_FREE_SPACE;
+
+        createStatisticTable(sheet, rowStart, colStart, teamList, FootballStatisticType.RED_CARDS,
+                borderCellStyle, fillingTableCellStyle);
+        createStatisticTable(sheet, rowStart, secondTableColStart, teamList, FootballStatisticType.CLEAN_SHEETS,
+                borderCellStyle, fillingTableCellStyle);
+    }
+
+    private void createStatisticTable(Sheet sheet, int rowStart, int colStart, List<Team> teamList,
+                                      FootballStatisticType type, CellStyle borderCellStyle, CellStyle fillingTableCellStyle) {
+
+
+        crateColumnsHeaders(sheet, rowStart, colStart, type, borderCellStyle);
+        rowStart++;
+
+        int place = 1;
+        Map<Long, Integer> topTen = getTopTenPlayersInSpecificStatistic(teamList, type);
+
+        for (Long playerId : topTen.keySet()) {
+            FootballPlayer player = playerRepository.findById(playerId).orElseThrow(
+                    () -> {
+                        throw new NoSuchElementException("Can't find player with given id");
+                    }
+            );
+            Row row = sheet.getRow(rowStart);
+
+            Cell cell = row.createCell(colStart);
+            cell.setCellValue(place++);
+            cell.setCellStyle(fillingTableCellStyle);
+
+            cell = row.createCell(colStart + 1);
+            cell.setCellValue(player.getFirstName() + " " + player.getLastName());
+            cell.setCellStyle(fillingTableCellStyle);
+
+            cell = row.createCell(colStart + 2);
+            cell.setCellValue(player.getTeam().getName());
+            cell.setCellStyle(fillingTableCellStyle);
+
+            cell = row.createCell(colStart + 3);
+            cell.setCellValue(topTen.get(playerId));
+            cell.setCellStyle(fillingTableCellStyle);
```

Comment:
tutaj tez dziwne podejscie i znowu +1 +2 +3 nie wiadomo dlaczego. Trzeba to zrefactorowac tez jakos